src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -1,7 +1,23 @@
 <template>
 	<div
-		v-if="promoAmount && promoAmount > 0"
 		class="
+			tw-rounded
+			tw-bg-brand-50
+			tw-w-full
+			tw-py-2
+			tw-px-1
+			tw-mb-2
+			tw-flex-col
+			tw-flex
+			tw-justify-start
+			lg:tw-justify-between
+			lg:tw-align-center
+			"
+	>
+		<!-- Progress bar appears as long as there is promo credit -->
+		<div
+			v-if="(promoAmount && promoAmount > 0) && (isMatchingCorporateCampaign || !isMatchingCorporateCampaign)"
```

Comment:
Once the proper passing of props has been resolved as listed below, this should be `v-if="(promoAmount && promoAmount > 0) && !isMatchingCampaign"`